### PR TITLE
Pagination on business list

### PR DESCRIPTION
**Pull request by dgmdan**
Resolves BW-558

### CODE REVIEW

File: Bizyhood/Views/listings/index.php [hunk 1]
```diff
@@ -18,6 +18,7 @@
     <?php dynamic_sidebar('businesses-right-sidebar') ?>
 </div>
 
+<?php echo paginate_links($pagination_args); ?>
```
Comment (eugenebiz) [inline Bizyhood/Views/listings/index.php]: Forgive my ignorance, is echo intended as a standard output mechanism, or is this a comment?

File: Bizyhood/Core.php [hunk 2]
```diff
@@ -756,9 +756,14 @@ public function businesses_shortcode($attrs)
     {
         $response = wp_remote_retrieve_body( wp_remote_get( "http://127.0.0.1:4567/businesses" ) );
         $response_json = json_decode($response);
+        $pagination = $response_json->pagination;
         $businesses = $response_json->data;
+        $pagination_args = array(
```
Comment (eugenebiz) [inline Bizyhood/Core.php]: When we return pagination information from the actual API, we would change this structure to match?

File: Bizyhood/Views/listings/index.php (hunk 1 above)
Comment (dgmdan) [inline Bizyhood/Views/listings/index.php]: Yeah, it's standard output.

File: Bizyhood/Core.php (hunk 2 above)
Comment (dgmdan) [inline Bizyhood/Core.php]: Yep.

**Comment by eugenebiz** [comment]
Ship it.

**Pull request merged**